### ISSUE TITLE
RSVP date is rendered incorrectly

### ISSUE THREAD

**Issue by mslourens** [Foundry376/Mailspring#1546]
The appointment date of the inline RSVP is incorrect. 
This is the date in the .ics:
```
DTSTART;TZID=W. Europe Standard Time:20190626T140000
DTEND;TZID=W. Europe Standard Time:20190626T150000
```

This is the rendered version in Mailspring:
![image](https://user-images.githubusercontent.com/1907152/60097215-393eaf80-9753-11e9-85c4-c7087d5f7a35.png)

This is the rendered version in gmail.com:
![image](https://user-images.githubusercontent.com/1907152/60097282-56737e00-9753-11e9-90bd-0d953be80f87.png)
Wed Jun 26, 2019 2pm – 3pm (CEST)

As you can see the date is off by not just a couple of hours, but by a whole day. It seems that this line is doing the calculation for the correct date: https://github.com/Foundry376/Mailspring/blob/33aba8f31fbbbcda924d335804801fcbb4e13f0d/app/internal_packages/events/lib/event-header.tsx#L108
It uses the timezone of my OS, which is set in Linux Mint to Central European Summer Time. But this could explain a difference of a couple of hours, not a whole day... 

I am using Linux Mint 19.1 and Mailspring 1.6.2 (installed by snap)

**Comment by bengotow** [comment]
Hey! Thanks for reporting this. This is super interesting—It looks like the date line is actually off by 19 days? I tried creating a calendar event with your exact DTSTART + DTEND lines, but surprisingly it's rendering that one correctly (same as Gmail.)

I think this one can't be timezone related because it's off by more than 24 hours. I tried enabling / disabling the "24 hour time" setting and that doesn't seem to do it either.

Will keep an eye out for this and see if I can narrow it down. It might be that this particular invite was broken in some other way? I'm curious if perhaps it included multiple DTSTART lines, and Mailspring picked up a previous recurrence of the event, maybe?

**Comment by deblike** [comment]
Hi!
Something similar happens to me, in my case the time shown in Mailspring is UTC time, doesn't use local time (UTC -03.00) to convert:

![Screenshot from 2019-11-26 14-18-09](https://user-images.githubusercontent.com/6434850/69657744-a4214900-1059-11ea-910a-7f8ca4fad6a3.png)

Same event as seen on Outlook 365

![Screenshot from 2019-11-26 14-31-01](https://user-images.githubusercontent.com/6434850/69657775-b3a09200-1059-11ea-96ee-e345523ef3de.png)

**Comment by glutio** [comment]
Wrong timezone in mailspring

Correct time in MSK timezone in webmail 
![hotmail com](https://user-images.githubusercontent.com/22550674/78716589-d310cb00-7927-11ea-8dbe-11080d6bd9c9.png)

Incorrect EST time in mailspring
![mailspring](https://user-images.githubusercontent.com/22550674/78716594-d60bbb80-7927-11ea-98de-58ac2fe4d271.png)

**Comment by tecoholic** [comment]
I have an example here https://github.com/Foundry376/Mailspring/issues/2039#issuecomment-657629873 to confirm as well.

**Comment by Phylu** [comment]
I am closing this issue as it should be resolved by #2093.
